### PR TITLE
[DOCS] 7.14.0 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #792. 

Preview [here](https://security-docs_811.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html).

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@jmikell821 I've removed the following PRs that are not Security PRs but were pulled in by the release notes tool:
- Adds support for fingerprint processor (#100541). 
- Remove external "Launch App Search" button (#100815).
- Heatmap / Swim lane integration (#97978).
- Updates help text for common repository settings (#97652).
- Collects all documents that fail to transform before stopping the migration (#96986).
- Provides guidance of "Custom" allocation behavior in ILM (#96111).
- Fixes remote clusters details flyout for long strings (#105592).
- Fixes some hardcoded translations on Dashboard & Discover (#105555).
- Fixes of the ja-JP.json not found error (#105023).
- Remove default value for error_distance param (#102222).
- Fixes default paramenter for ignore_missing in csv processor (#100316).
- Fixes assignment v comparison and null type check in metrics detail page (#95102).

**Review by dplumlee — APPROVED** [review]
looks good on the bugs section, i didn't work too much on the features this release

**Comment by cnasikas** [comment]
Hi @nastasha-solomon!

I am not familiar with the release notes 🙂 so sorry if I am mistaken. I have some questions:

1. Why the Swimlane connector is on the Bug & Enhancements category? Is it not consider a feature?
2. Why Cases privileges (RBAC) are not part of the list?

**Comment by nastasha-solomon** [comment]
Hey @cnasikas, not a problem, and thanks for reviewing these! The Swimlane PR that was flagged for release notes had the `release_note:enhancement` label, hence its current placement in the 7.14 draft. I can move it to the features section if that's a more appropriate place for it to go though. 

To address your second question, our release notes tool didn't pull in any PRs for the RBAC for Cases feature. I did some quick digging to see where that PR might be and found https://github.com/elastic/kibana/pull/95058. If this is the PR that should be release-noted for 7.14, it might've gotten overlooked by our tool because--although it has the appropriate team labels--the release notes label (`release_note:plugin_api_changes`) falls outside of the scope of release notes labels our tool looks for. AFAIK, our release notes tool only pulls in PRs that have specific Security team labels and release note labels for bugs, features, enhancements, deprecations, and breaking changes.

**Comment by jonathan-buttner** [comment]
Ah thanks for the heads up about the labels @nastasha-solomon

**Comment by cnasikas** [comment]
Thanks @nastasha-solomon for explaining the release note labels. It is difficult for me to understand which labels should I use every time.

About the Swimlane connector, I think is more appropriate to be on the Feature section as it is a new feature.

You are right about the RBAC. This is the right PR and we should put another release label also. I think RBAC should also be in the feature section because the user can create a role and set specific privileges (all/write/none) for Cases. It is a new capability that was not possible prior to 7.14. What do you think?

File: docs/release-notes.asciidoc [hunk 1]
```diff
@@ -7,6 +7,63 @@
...
+[[release-notes-7.14.0]]
+== 7.14.0
+
+[discrete]
+[[features-7.14.0]]
+==== Features
+* Host isolation allows analysts to isolate hosts from their networks while investigating a potential attack. Analysts can use this feature to respond to malicious activity by containing infected hosts, curbing potential attacks, and preventing lateral movement to other hosts. This feature is supported on Windows and macOS.
+* Adds malware protection for Linux endpoints. Users can enable Linux malware protection in their policy to receive detection alerts. ({pull}103404[#103404]) ({pull}95014[#95014])({pull}104984[#104984]).
+* Adds threat intelligence to alerts ({pull}101553[#101553])({pull}103383[#103383]).
+* New functionality and usability improvements were added to the Osquery Manager integration:
```
Comment (joepeeples) [inline docs/release-notes.asciidoc:20]:
```suggestion
* Adds new functionality and usability improvements to the Osquery Manager integration:
```

File: docs/release-notes.asciidoc [hunk 2]
```diff
@@ -7,6 +7,63 @@
...
+[[bug-fixes-7.14.0]]
+==== Bug fixes and enhancements
+* Adds a merge strategy key to `kibana.yml` and adds additional security keys to the Docker container that Elastic Security previously overlooked ({pull}103800[#103800]).
+* Adds an overflow container to the rule name column in the Exceptions table for exceptions that have been assigned to three or more rules ({pull}103377[#103377]).
+* Adds the option to select all rules within the Rules table that match the currently selected filter ({pull}100554[#100554]).
+* Adds the Threat Intelligence card to the Overview page ({pull}100423[#100423]).
+* Introduces the Swimlane connector for rules and cases ({pull}100086[#100086]).
+* Enhances the callout that describes missing privileges and feature access ({pull}98125[#98125]).
+* Fixes the rule preview issue that occurred if users created a threshold rule that was configured to group the IP data type ({pull}105126[#105126]).
+* Removes the comma delimiter for the `is one of operator` when defining rule exception conditions ({pull}104960[#104960]).
```
Comment (joepeeples) [inline docs/release-notes.asciidoc:41]:
```suggestion
* Removes the comma delimiter for the `is one of` operator when defining rule exception conditions ({pull}104960[#104960]).
```

File: docs/release-notes.asciidoc [hunk 3]
```diff
@@ -7,6 +7,63 @@
...
+* Resolves bug that left outdated validation messages on the action type selection form ({pull}104868[#104868]).
+* Fixes the sort logic that didn't work for certain fields within the Rules table ({pull}103960[#103960]).
+* Allows activity log scrolling on small screens ({pull}103852[#103852]).
+* Fixes the bug that cause the checkbox value for *Show only threat indicator alerts* from updating properly within the Alerts table ({pull}103746[#103746]).
```
Comment (joepeeples) [inline docs/release-notes.asciidoc:45]:
```suggestion
* Fixes the bug that caused the checkbox value for *Show only threat indicator alerts* from updating properly within the Alerts table ({pull}103746[#103746]).
```

File: docs/release-notes.asciidoc [hunk 4]
```diff
@@ -7,6 +7,63 @@
...
+* Disables the *Load Elastic prebuilt rules and timeline templates* button when pre-built rules are loading ({pull}103568[#103568]).
+* Allows users to view the details of a deleted rule ({pull}103491[#103491]).
+* Includes actions and responses for endpoints only ({pull}103159[#103159]).
+* Resolves the issue that cause an error message to display if users created rule exceptions with empty fields ({pull}102583[#102583]).
```
Comment (joepeeples) [inline docs/release-notes.asciidoc:49]:
```suggestion
* Resolves the issue that caused an error message to display if users created rule exceptions with empty fields ({pull}102583[#102583]).
```

File: docs/release-notes.asciidoc [hunk 5]
```diff
@@ -7,6 +7,63 @@
...
+* Removes the search bar on the *Activity log* tab ({pull}102550[#102550]).
+* Does not show activity log error popups ({pull}102450[#102450]).
+* Shows up to one hour of relative time in the activity log when viewing it from the endpoint details flyout. ({pull}102162[#102162]).
+* Updates mappings for detection alerts to ECS v1.10.0 ({pull}101680[#101680]).
+* Fixes timestamp bugs within source indexes when the formats are not in ISO 8601 format ({pull}101349[#101349]).
+* Exposes the EQL query in Kibana logs for detections ({pull}100565[#100565]).
+* Resolved bugs linked to invalid KQL queries ({pull}99442[#99442]).
```
Comment (joepeeples) [inline docs/release-notes.asciidoc:56]:
```suggestion
* Resolves bugs linked to invalid KQL queries ({pull}99442[#99442]).
```

**Review by joepeeples — COMMENTED** [review]
Just a few small line-edits, looks good!

**Comment by peluja1012** [comment]
Hi @nastasha-solomon, could we please add a "Known Issue" for this [bug](https://github.com/elastic/kibana/issues/106233#issuecomment-888803499). We could say something like:


When upgrading from 7.8 to 7.14, rule execution and activation/deactivation of rules could fail (https://github.com/elastic/kibana/issues/106233). To work around this issue you can use the [PATCH rule API](https://www.elastic.co/guide/en/security/current/rules-api-update.html) to update each rule that presents this problem. The payload of the PATCH call should set the `author` field to `[]` as follows. After the `author` field is populated the rule will start to work as expected.
```
PATCH <kibana host>:<port>/api/detection_engine/rules
{
  "id": <id-value-of-rule>",
  "author": []
}
```

**Review by peluja1012 — APPROVED** [review]
LGTM. Thanks for including the last minute "known issue"!

File: docs/release-notes.asciidoc [hunk 6]
```diff
@@ -7,6 +7,80 @@
...
 
+[discrete]
+[[release-notes-7.14.0]]
+== 7.14.0
+
+[discrete]
+[[features-7.14.0]]
+==== Features
+* Host isolation allows analysts to isolate hosts from their networks while investigating a potential attack. Analysts can use this feature to respond to malicious activity by containing infected hosts, curbing potential attacks, and preventing lateral movement to other hosts. This feature is supported on Windows and macOS.
+* Adds malware protection for Linux endpoints. Users can enable Linux malware protection in their policy to receive detection alerts. ({pull}103404[#103404]) ({pull}95014[#95014])({pull}104984[#104984]).
```
Comment (joepeeples) [inline docs/release-notes.asciidoc:18]:
```suggestion
* Adds malware protection for Linux endpoints. Users can enable Linux malware protection in their policy to receive detection alerts. ({pull}103404[#103404])({pull}95014[#95014])({pull}104984[#104984]).
```

**Pull request merged**